.github/workflows/release_version.yaml

```diff
@@ -0,0 +1,128 @@
+name: Release tag version
+
+on:
+  workflow_dispatch:
+    branches:
+      - feature/*
+    inputs:
+      version:
+        required: true
+      description:
+        required: true
+jobs:
+  build-test:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - uses: actions/setup-go@v2
+        with:
+          go-version: '1.17'
+
+      - name: Setup project
+        run: make clean
+
+      - name: Tests - Unit
+        run: make test
+
+  release:
+    runs-on: ubuntu-latest
+    needs: build-test
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Get credentials
+        uses: leanix/secrets-action@master
+        with:
+          secret-store-credentials: ${{ secrets.INJECTED_SECRET_STORE_CREDENTIALS }}
+
+      - uses: actions/setup-go@v2
+        with:
+          go-version: '1.17'
+
+      - name: Login to Azure Container Registry
+        uses: azure/docker-login@v1
+        with:
+          login-server: 'leanixacr.azurecr.io'
+          username: ${{ env.ACR_USERNAME }}
+          password: ${{ env.ACR_PASSWORD }}
+
+      - name: Validate version input
+        run: echo ${{ github.event.inputs.version }}
+
+      - name: Extract base version
+        env:
+          SEM_VERSION: ${{ github.event.inputs.version }}
+        run: |
+          IFS="."
+          read -ra base_version <<< "$SEM_VERSION"
+          echo "BASE_VERSION=$base_version" >> $GITHUB_ENV
+
+      - name: Create github tag for semantic version
+        uses: rickstaa/action-create-tag@v1
+        with:
+          tag: ${{ github.event.inputs.version }}
+          message: ${{ github.event.inputs.description }}
+      - name: Create/Update github tag for latest version
+        uses: rickstaa/action-create-tag@v1
+        with:
+          tag: ${{ env.BASE_VERSION }}.0.0-latest
```

Comment:
shouldn't  it be  `${{ env.BASE_VERSION }}-latest` ? Also I dint understand why a tag with `-latest` is necessary?

Reply:
You are right, I removed the tag creation. it was not necessary